tests/SprykerEcoTest/Zed/Heidelpay/Business/HeidelpayFacadeIsCaptureSuccessTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Heidelpay\Business;
+
+use Generated\Shared\Transfer\PaymentTransfer;
+use Orm\Zed\Sales\Persistence\SpySalesOrder;
+use SprykerEco\Zed\Heidelpay\Persistence\HeidelpayQueryContainer;
+use SprykerEco\Zed\Heidelpay\Persistence\HeidelpayQueryContainerInterface;
+use SprykerEcoTest\Zed\Heidelpay\Business\DataProviders\PaymentBuilder;
+use SprykerEcoTest\Zed\Heidelpay\Business\DataProviders\Transaction\CaptureTransactionTrait;
+
+/**
+ * @group Functional
+ * @group SprykerEcoTest
+ * @group Zed
+ * @group Heidelpay
+ * @group Business
+ * @group HeidelpayFacadeIsCaptureSuccessTest
+ */
+class HeidelpayFacadeIsCaptureSuccessTest extends HeidelpayPaymentTest
+{
+    use CaptureTransactionTrait;
+
+    /**
+     * @return void
+     */
+    public function testCheckShouldReturnFalseWhenCaptureTransactionLogIsEmpty(): void
```

Comment:
Please rename the test case according to the following pattern `test*FacadeMethodName*Expectation*`. 
`testIsCaptureSuccessfulShouldReturnFalseWhenCaptureTransactionLogIsEmpty`.